contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -906,6 +906,31 @@ class tsqlCommonMutator : public TSqlParserBaseListener
 	void exitFunction_call(TSqlParser::Function_callContext *ctx) override
 	{
 		handleGeospatialFunctionsInFunctionCall(ctx);
+
+		if (ctx->func_proc_name_server_database_schema())
```

Comment:
Is this additional handling for `CREATE-VIEW` statements? if Yes, then do we have adequate tests for this scenario?

Reply:
Yes, this additional handling is for `CREATE-VIEW` statements. We already have adequate tests added for views that uses `TRIM` function.

Reply:
I am not able to understand this properly, will it replace all the occurrences of TRIM with sys.trim?

Reply:
Yes, without above handling TRIM was not getting replaced with sys.TRIM, when trim is called inside VIEW/Function/Procedure creation.